supermarq-benchmarks/supermarq/benchmark.py

```diff
@@ -34,15 +30,6 @@ def cirq_circuit(self) -> cirq.Circuit | list[cirq.Circuit]:
         """
         return self.circuit()
 
-    def qiskit_circuit(self) -> qiskit.QuantumCircuit | list[qiskit.QuantumCircuit]:
```

Comment:
Put this back. Bring `qiskit_qss` HPCA tutorial.

Reply:
I'm not going to put this back in order to justify putting the converters in Supermarq. Not worth it.